tests/modules/programs/carapace/basic-configuration.nix

```diff
@@ -0,0 +1,13 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    programs.bash.enable = true;
+    programs.carapace.enable = true;
```

Comment:
The package needs to be stubbed so as not to be downloaded.
See other tests for examples of using `nmt.stubs`